packages/remix/src/utils/utils.ts

```diff
@@ -32,14 +36,19 @@ export function assertValidClerkState(val: any): asserts val is ClerkState | und
  * @returns string
  */
 export const getEnvVariable = (name: string, context: AppLoadContext | undefined): string => {
+  // Remix + Cloudflare pages
+  if (typeof (context?.cloudflare as CloudflareEnv)?.env !== 'undefined') {
```

Comment:
Would be nice to have everything in one place so we can move those lines together with the rest of Cloudflare Pages check.

Reply:
We probably don't need the `typeof` check here as this is testing a local identifier and not the global (`window`/`globalThis`) namespace but everything else makes sense to me :)